cas-server-core-configuration/src/main/java/org/apereo/cas/configuration/model/support/wsfed/WsFederationProperties.java

```diff
@@ -0,0 +1,98 @@
+package org.apereo.cas.configuration.model.support.wsfed;
+
+import org.apereo.cas.configuration.model.core.authentication.PersonDirPrincipalResolverProperties;
+import org.springframework.boot.context.properties.NestedConfigurationProperty;
+
+/**
+ * This is {@link WsFederationProperties}.
+ *
+ * @author Misagh Moayyed
+ * @since 5.0.0
+ */
+
+public class WsFederationProperties {
+
+    private String identityAttribute = "upn";
+    private String identityProviderIdentifier = "https://adfs.example.org/adfs/services/trust";
+    private String identityProviderUrl = "https://adfs.example.org/adfs/ls/";
+    private String signingCertificateResources = "classpath:adfs-signing.crt";
+    private String relyingPartyIdentifier = "urn:cas:localhost";
+    private int tolerance = 10000;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ['10000' is a magic number.](https://www.codacy.com/app/mmoayyed/cas/file/2769871230/issues/source?bid=3256115&fileBranchId=3357337#l20)